### ISSUE TITLE
Data Science with Python : Exploratory Data Analysis using Python

### ISSUE THREAD

**Issue by prathimacode-hub** [girlscript/winter-of-contributing#348]
Welcome to 'DSWP' Team, good to see you here

This issue will helps readers in gaining all the guidance that one needs to know about Exploratory Data Analysis using Python. Tutorial to Exploratory Data Analysis using Python and how it's applied using sample code.

To get assigned to this issue, add your serial numbers mentioned in the spreadsheet of "Data Science with Python", the approach one would follow and choice you prefer (Documentation, Audio, Video). You can go with all three or any number of options you're interested to work on.

If you had referred any resources, add them up in "DS Resources". Similarly if you had used datasets, include them in "DS Datasets".

Domain : Data Analysis and Visualization

Mentors Assigned : Shivang Dubey / Dhruv Bajaj

Points to Note :

The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.

"Issue Title" and "PR Title should be the same. Include issue number along with it.

Changes should be made inside the Datascience_With_Python/ directory & Datascience_With_Python branch.

Follow Contributing Guidelines & Code of Conduct before start Contributing.

This issue is only for 'GWOC' contributors of 'DSWP' domain.

All the best. Enjoy your open source journey ahead. 😎

**Comment by nakshatra-garg** [comment]
I would like to work on this issue

**Comment by developer-metalhead** [comment]
Hi @prathimacode-hub I would like to be assigned this issue

**Comment by GopinathBalaji** [comment]
Hello, I am Gopinath Balaji with serial number:961. I would like to do documentation on this issue.

**Comment by prathimacode-hub** [comment]
Kindly provide all the details to assign it up. Approach is missing and whoever states it first would get this issue assigned. @GopinathBalaji @developer-metalhead @nakshatra-garg

**Comment by GopinathBalaji** [comment]
I have mentioned that I would like to work on the documentation part.

**Comment by ayushijindal** [comment]
I would like to work on this issue
issue title: Exploratory Data Analysis using Python #348
ID:253
documentation format

**Comment by prathimacode-hub** [comment]
> I have mentioned that I would like to work on the documentation part.

Yes, approach in the sense how would you like to proceed with an issue. @GopinathBalaji

**Comment by nakshatra-garg** [comment]
https://colab.research.google.com/drive/1gP-rr8RPeUuo1Jjwr0gVXWJC6Vkcnzqj?usp=sharing
I can work on this issue by explaining this notebook. It uses retail data and has analysis and visualiztions

**Comment by ayushijindal** [comment]
> Kindly provide all the details to assign it up. Approach is missing and whoever states it first would get this issue assigned. @GopinathBalaji @developer-metalhead @nakshatra-garg

I would like to work on this issue
issue title: Exploratory Data Analysis using Python #348
ID:253
Documentation format

**Comment by developer-metalhead** [comment]
I will write the different steps and approaches of EDA. I will also add the code to the documentation.
I am Kumar Satwik with serial number:160

**Comment by GopinathBalaji** [comment]
> Kindly provide all the details to assign it up. Approach is missing and whoever states it first would get this issue assigned. @GopinathBalaji @developer-metalhead @nakshatra-garg

I would like to explain the process of EDA in detail. To name a few, the steps that I would include are: getting the dataset, data cleaning, deal with missing values, etc.

**Comment by Adi7171** [comment]
I am Aditya serial no: 92 I would take up a dataset from kaggle and  then I would do appropriate changes to data using numpy ,pandas and then visualize it using matpplotlib and write comments regarding it

**Comment by subhendudash02** [comment]
Hi @prathimacode-hub I would like to work on this issue.
Serial number: 297
Documentation format
Issue: Exploratory Data Analysis using Python #348

**Comment by ashish-reddy-20-08** [comment]
Hello, I am M. Ashish Reddy with serial number:174. I would like to do documentation on this issue.
My approach would be the various steps in EDA like outlier analysis, missing values, and the first-moment business decision and then at last I would also be discussing feature engineering.

**Comment by deepthi1107** [comment]
Hello, I am Deepthi M with a serial number:172. I would like to do an audio presentation on this issue. I will add how important to do EDA and related code to EDA

**Comment by vanshhhhh** [comment]
Hello, I would like to contribute in the documentation of this topic. My approach would be to analyze datasets to summarize their main characteristics, often with visual methods. I will cover Data Collection, Data Cleaning, Data Preprocessing, Data Visualization.

Vansh Sharma (Serial Number :  680, DS-Python Batch 18)

**Comment by prathimacode-hub** [comment]
@deepthi1107 @ashish-reddy-20-08, you both already have an issue assigned. You can work on only 1 issue at a time.
@nakshatra-garg, sorry to say but I couldn't open your link provided. @ayushijindal, next on queue, documentation part is assigned to you. Others, if you're into Audio/Video. Let me know to get yourself assigned.

If you'couldn't grab an issue, they are plenty number of issues arriving. Don't get disheartened.

**Comment by ayushijindal** [comment]
> @deepthi1107 @ashish-reddy-20-08, you both already have an issue assigned. You can work on only 1 issue at a time.
> @nakshatra-garg, sorry to say but I couldn't open your link provided. @ayushijindal, next on queue, documentation part is assigned to you. Others, if you're into Audio/Video. Let me know to get yourself assigned.
> 
> If you'couldn't grab an issue, they are plenty number of issues arriving. Don't get disheartened.

THANK YOU for assigning me this issue

**Comment by rajveer43** [comment]
I would like to work on this issue 
Batch 15

**Comment by nakshatra-garg** [comment]
I've made a video on exploratory data analysis in the past. Will this video mentioned below will be fine?
https://www.youtube.com/watch?v=q6oCG1aNVk8

**Comment by AnnikaD0104** [comment]
I would like to work in this issue serial no 147

**Comment by ayushijindal** [comment]
update on documentation 
Serial no: 253
I am using the breast cancer dataset to perform EDA and soon the documentation will be ready
mentor:@DhruvBajaj01 / @ShivangDubey

**Comment by Aaksha-sharma** [comment]
I want to contribute in this issue

**Comment by shivangdubey** [comment]
> update on documentation
> Serial no: 253
> I am using the breast cancer dataset to perform EDA and soon the documentation will be ready
> mentor:@DhruvBajaj01 / @shivangdubey

Do add the resources in the Resource folder.

**Comment by DhruvBajaj01** [comment]
I will be mentoring and reviewing this issue

**Comment by shripaddhopate** [comment]
I would be glad to work on this issue.
Name : Shripad Dhopate
Batch : 21
Type : Documentation , Video
Serial No: 930

**Comment by Apoorv-17** [comment]
Hello, I am Apoorv Yadav with serial number: 537. I would like to do documentation on this issue.

**Comment by prathimacode-hub** [comment]
> I've made a video on exploratory data analysis in the past. Will this video mentioned below will be fine?
> https://www.youtube.com/watch?v=q6oCG1aNVk8
> 
> @prathimacode-hub

That's awesome but can you do it again? If you are upto, I shall assign you for video? @nakshatra-garg

**Comment by nakshatra-garg** [comment]
Yes, I can do it again. @prathimacode-hub

**Comment by Ruchika-20** [comment]
I would love to work on this issue. My approach regarding this issue will be description of the issue which will include definition, real world applications , code and all other necessary explaination part.
Name : Ruchika Singh
Batch : 10
Type : Documentation
Serial No: 331

**Comment by prathimacode-hub** [comment]
Issue assigned to @nakshatra-garg for video contribution
